DOCS/man/options.rst

```diff
@@ -7454,13 +7454,14 @@ Miscellaneous
 
     This is a string list option. See `List Options`_ for details.
 
-``--cover-art-whitelist=<no|yes>``
-    Whether to load files with a filename among "AlbumArt", "Album", "cover",
-    "front", "AlbumArtSmall", "Folder", ".folder", "thumb", and an extension in
-    ``--cover-art-auto-exts``, as cover art. This has no effect if
-    ``cover-art-auto`` is ``no``.
+``--cover-art-whitelist=filename1,filename2,...``
```

Comment:
This is breaking change.

Reply:
Hence why it's added to interface changes, also I doubt anyone was disabling this specifically instead of setting `--audio-display=no`.

Reply:
We can rename it to `--cover-art-filenames` if you prefer.

Reply:
I really don't think backwards compatibility is that important here, this isn't `loadfile`, it's an obscure option I added 2 years ago and it was only a way to separate it from `--cover-art-auto=fuzzy` (https://github.com/mpv-player/mpv/pull/9550) and to document what cover art filenames are loaded rather than for it to be used. Someone who doesn't want to load cover art would set `--audio-display=no` to disable it completely as recommended by the terminal output.

Reply:
Also `--cover-art-whitelist=no` would still disable this as long as you don't have files like `no.jpg` in music directories.

Reply:
My point is that we are struggling to acknowledge what "change" and "deprecate" means in mpv. I don't mind breaking compatibility, it is not like people cannot update their config. But in the same time in other cases we try to "pretend" that backward compatibility matters. We should be consistent, imho.